client/src/components/layouts/rtl-layout.css

```diff
@@ -6,6 +6,14 @@ Increase the spacing in paragraphs
   line-height: 2rem;
 }
 
+/* 
+Profile section
+*/
+
+[dir='rtl'] :is(.media-left, .media > .pull-left) {
```

Comment:
Let's remove .media > .pull-left and turn it into `[dir='rtl'] .media-left`.
Also let's make it more specific to the Portfolio page.

Reply:
Couldn't find a good hook, so I had to add an `id`.